src/index.js

```diff
@@ -23,6 +23,32 @@ const moduleName =
 	moduleAliases[packageName] ||
 	packageName.replace('resin-', '').replace('balena-', '');
 
+function getCodeowners() {
+	// git ls-files '*CODEOWNERS'
+	// * @user1 @user2
+	let codeownerSet = new Set();
+	const codeownersFiles = gitMultilineResults('ls-files *CODEOWNERS');
+	codeownersFiles.forEach(function(file) {
+		codeownerLine = fs
+			.readFileSync(file)
+			.toString()
+			.toLowerCase()
+			.split('\n')
+			.filter(n => n)
+			.filter(n => !n.startsWith('#'));
```

Comment:
Let's combine the two filters:
```suggestion
			.filter(n => n && !n.startsWith('#'));
```